src/components/date-picker/demos/index.tsx

```diff
@@ -166,6 +209,9 @@ export default () => {
       <DemoBlock title='自定义每列的渲染内容'>
         <CustomRender />
       </DemoBlock>
+      <DemoBlock title='周 - 工作日选择器'>
```

Comment:
```suggestion
      <DemoBlock title='周选择器'>
```
就叫“周选择器”吧，工作日是指周一到周五的

Reply:
hhh欧克